Translations/ko.lproj/preferences.strings

```diff
@@ -153,10 +153,10 @@
 "453.title" = "없음";
 
 /* Class = "NSButtonCell"; title = "Enable Notification Center integration"; ObjectID = "SOL-6U-5P9"; */
-"SOL-6U-5P9.title" = "Enable Notification Center integration";
+"SOL-6U-5P9.title" = "macOS 알림 센터 사용";
 
 /* Class = "NSButtonCell"; title = "Use unified toolbar (resets customizations)"; ObjectID = "b5e-bj-Pgz"; */
-"b5e-bj-Pgz.title" = "Use unified toolbar (resets customizations)";
+"b5e-bj-Pgz.title" = "통합된 툴바 사용 (설정 초기화됨)";
```

Comment:
I have no knowledge of Korean, but google translator tells me this says "integrated". Note that is unified but not integrated, since it unifies the toolbar and the title bar.